src/pkg/utils/utils.go

```diff
@@ -0,0 +1,283 @@
+package utils
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"os/user"
+	"regexp"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/containers/toolbox/pkg/podman"
+	"github.com/shirou/gopsutil/host"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+)
+
+const (
+	idTruncLength = 12
+)
+
+// GetCgroupsVersion returns the cgroups version of the host
+//
+// The information is taken from the output of `podman info` command
+func GetCgroupsVersion() (string, error) {
+	podmanInfo, err := podman.PodmanInfo()
+	if err != nil {
+		return "", err
+	}
+	cgroupVersion := fmt.Sprint(podmanInfo["host"].(map[string]interface{})["CgroupVersion"])
```

Comment:
Podman 1.9.0 has changed the JSON key to `cgroupVersion`. :/

I think we will be better off using code similar to what Podman itself uses to check the Cgroups version. @mheon pointed me to `IsCgroup2UnifiedMode` in `libpod/pkg/cgroups`.